nilearn/glm/first_level/first_level.py

```diff
@@ -900,7 +900,7 @@ def fit(
             self._reporting_data["run_imgs"][run_idx] = {}
             if isinstance(run_img, (str, Path)):
                 self._reporting_data["run_imgs"][run_idx] = (
-                    parse_bids_filename(run_img)
+                    parse_bids_filename(run_img, legacy=False)
```

Comment:
Just trying to understand, would this mean that if someone has older BIDS file structure our GLM object won't be able to detect those files?

Reply:
nope 
the legacy in this case refers to the structure of the output (the name of the keys in the output dict), not the input

Reply:
ah ok